gemrb/GUIScripts/pst/GUIINV.py

```diff
@@ -406,36 +412,97 @@ def OnDragItem ():
 			GemRB.SetGlobal("APPEARANCE","GLOBAL",0)
 
 	UpdateInventoryWindow ()
+	return
+
+# Duplicated from InventoryCommon and should be removed once behaviour is consolidated
+def DecreaseStackAmount ():
+	"""Decreases the split size."""
+
+	Text = ItemAmountWindow.GetControl (6)
+	Amount = Text.QueryText ()
+	number = int ("0"+Amount)-1
+	if number<1:
+		number=1
+	Text.SetText (str (number))
+	return
 
+# Duplicated from InventoryCommon and should be removed once behaviour is consolidated
+def IncreaseStackAmount ():
+	"""Increases the split size."""
+
+	Text = ItemAmountWindow.GetControl (6)
+	Amount = Text.QueryText ()
+	number = int ("0"+Amount)+1
+	if number>StackAmount:
+		number=StackAmount
+	Text.SetText (str (number))
+	return
+
```

Comment:
Its the ItemAmountWindow.GetControl that causes an error, if I use InventoryCommon.IncreaseStackAmount, it looks for the ItemAmountWindow there, whereas it is actually loaded in pst/GUIINV.py line 503

Although looking at it again, it is probably an easy canditate for consolidation, I think I might be able to actually end up with less rather than more here, I will try again tonight.

Reply:
you can easily get rid of `ItemAmountWindow` altogether by using `AddAlias` and `GetView` instead of a global variable.